storage/redis_cluster.go

```diff
@@ -361,6 +363,7 @@ func (r *RedisCluster) GetKeys(filter string) []string {
 	filterHash := ""
 	if filter != "" {
 		filterHash = r.hashKey(filter)
+		filterHash = base64.StdEncoding.EncodeToString([]byte(fmt.Sprintf(`{"org":"%s"`, filterHash)))
```

Comment:
We should make it backward compatible way, with keys which was generated with old format (e.g. murmur32 hash), so we should have both old and new filter. 
I propose to move this code directly to Keys List API method, which will look like:
```
orgIDB64 := fmt.Sprintf(`{"org":"%s"`, filterHash)
keys := storage.GetKeys(orgID)
keys = append(keys, orgIDB64...)